man/paru.8

```diff
@@ -70,6 +70,11 @@ Paru will also clean cached AUR packages and any untracked files in the
 cache. Cleaning untracked files will wipe any downloaded sources or
 built packages but will keep already downloaded VCS sources.
 
+.TP
+.B \-Sss
```

Comment:
I'd rather just say it gives some extra verbose output. I feel what -Sss does may get added to over time so may as well keep it generic.